src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java

```diff
@@ -45,6 +45,8 @@
  * <p>
  * The save operation is loaded off of the GUI thread using {@link BackgroundTask}. Callers can query whether the
  * operation was canceled, or whether it was successful.
+ * However, the callee must make sure that methods are called from the JavaFX application thread as {@link BasePanel}
```

Comment:
What does "callee" mean here? This class? Doesn't it contradict, that the change above ensures that it is in the JavaFX app thread. In case, it is a typo (and caller is meant), what does "methods" mean here? (Should it be "this methos is"?)